src/branding/centos/Makefile.am

```diff
@@ -4,6 +4,9 @@ centosbranding_DATA = \
 	src/branding/centos/branding.css \
 	$(NULL)
 
+EXTRA_DIST += $(centosbranding_DATA)
+CLEANFILES += $(centosbranding_DATA)
```

Comment:
None of these files should be cleaned, they are original source.

It seems odd that they would need to go to EXTRA_DIST -- automake dists every file in _DATA etc. already anyway.